docs/DataWorkflows/FetchingYourData.md

```diff
@@ -246,6 +243,212 @@ Critically, the reference to the EXACT version of the file you used in your
 analysis will be fetched if you need to
 [access the data again](#clone-the-static-data).
 
+# Unzipping your data
+
+Data on PMACS might be zipped, or unzipped. In the case of unzipped data, you should be good to go following the guidelines on finding files you need [here](./FetchingYourData.md)! If your data is zipped, you might need a little more work.
+
+The following walkthrough should help you unzip your data. These lines of code can be customized per your needs to unzip some or all subjects and then some or all files: 
+
```

Comment:
similar to the above organization I proposed, here I'd add a subsection: 

### Figuring out what data is available (single participant)